Benchmarking/GC_stub/RnD1system_benchmarking_analysis_GCstub.py

```diff
@@ -0,0 +1,259 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Feb  2 09:30:01 2022.
+
+@author: AnnaWiniwarter
+"""
+
+import numpy as np
+import ixdat
+
+from pathlib import Path
+from matplotlib import pyplot as plt
+from ixdat.techniques.ec import ECMeasurement
+
+
+# -----------------------EDIT SETTINGS HERE----------------------------------
+# Settings for choosing which part of the script is executed:
+DATA_SOURCE = "ixdat"
+# DATA_SOURCE can be "raw" (for importing EC and MS data from Zilien .tsv file),
+# or "ixdat" (for importing ixdat .csv files as generated when using setting "raw")
+# in either case, for plotting the CV only, the biologic CVA file is imported
+# automatically
+WHICH_PART = "plot+fit_HER"
+# WHICH_PART can be "plot_CVs", "plot+fit_HER", "plot+fit_gas_exchange"
+
+SAVE_FIGURES = True
+# SAVE_FIGURES set to True: figures will be saved automatically.
+FIGURE_TYPE = ".png"
+# FIGURE_TYPE can be any format matplotlib accepts for saving figures, eg.
+# ".png", ".svg", ".pdf" etc.
+
+# select the directory of the raw data.
+EXP_NAME = "RnD1system_benchmark_GC"
+THIS_DIR = Path(__file__).parents[1].resolve()
+DATA_DIRECTORY = THIS_DIR / "data"
+
+# select the filenames of the raw data.
+ZILIEN_FILENAME = "2022-03-10 08_39_41 GC benchmarking2.tsv"
+BIOLOGIC_FILENAME = "2022-03-10 08_39_41 GC benchmarking2_09_01_CVA_DUSB0_C01.mpt"
+
+
+# In addition to the above general settings, when using a diffent dataset,
+# the time of when the different measurements occur in the dataset needs to be
+# adjusted in by changing the "tspan" in each section accordingly. To this end
+# it is recommended to use full_data.plot_measurement(tspan=[t1,t2]) after importing
+# and varying t1 and t2 to find the tspans of interest. (see README.md)
+# ----------------------- END OF EDIT SETTINGS ------------------------------
+
+
+# ----------- functions for exponential fitting of decay --------------------
+# for now copied from here: https://stackoverflow.com/questions/3938042/fitting-exponential-decay-with-no-initial-guessing
```

Comment:
how do I do this properly?

Reply:
Oh, I actually just meant to remove the "for now" as that suggests that it will change :D Fitting an exponential seems like a simple enough task that we should be able to evaluate that the approach you found on SO is valid, so I consider that an ok reference.